### PR TITLE
DHFPROD-4709: Show correct source format for Load steps in flow

### PR DESCRIPTION
**Pull request by wooldridge**
### Description

Source format now correctly displayed in flow steps:

Screen Shot 2020-04-27 at 11 58 28 AM

The fix was actually added with this:

https://github.com/marklogic/marklogic-data-hub/pull/3876

This commit adds tests for default (JSON) and non-default (XML) steps.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/config/bench.config.ts [hunk 1]
```diff
@@ -216,6 +216,106 @@ const jobRespSuccess = {
...
+      "1": {
+        "name": "loadJSON",
+        "description": "",
+        "options": {
+          "sourceQuery": null,
+          "collections": ["loadJSON"],
+          "loadData": {
+            "name": "loadJSON"
+          },
+          "outputFormat": "json",
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/config/bench.config.ts:238]: @wooldridge can you pls with latest develop if a flow artifact stores output format. Mine doesnt seem to.

Comment (bsrikan) [reply marklogic-data-hub-central/ui/src/config/bench.config.ts:238]:
```
"1": {
  "name": "loadCustomer", 
  "options": {
    "loadData": {
      "name": "loadCustomer"
    }
  }, 
  "stepDefinitionName": "default-ingestion", 
  "stepDefinitionType": "INGESTION"
}
```

Comment (srinathgit) [reply marklogic-data-hub-central/ui/src/config/bench.config.ts:238]: @bsrikan is right. It is in settings doc not in the flow

Comment (bsrikan) [reply marklogic-data-hub-central/ui/src/config/bench.config.ts:238]: Also these details might change depending how loadData is rewritten. To keep it simple, maybe we can just test with an xml load data now to keep refactoring to a minimum in the future. Thoughts?

Comment (srinathgit) [reply marklogic-data-hub-central/ui/src/config/bench.config.ts:238]: Yes, I think this can be revisited once loadData / settings is rewritten

**Comment by wooldridge** [comment]
@bsrikan and @srinathgit, thanks. I removed the outputFormat property in the test data and the JSON test for the Load step.

**Pull request merged**